# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/canvas/WebGLFramebuffer.h
# Commit: 7c94d37c446e
# Full Hash: 7c94d37c446ea0d996bf6e6e04eb66101e126b5c
# Author: Jeff Gilbert <jgilbert@mozilla.com>
# Date: 2020-06-11 09:34:54
# Regressor Bug: 1632249
# File Overlap Count: 1
# Description:
#   Bug 1632249 - WebXR compositing fixes. r=imanol
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D78117
# ==============================================================================

diff -r 204b899f436d -r 7c94d37c446e dom/canvas/WebGLFramebuffer.h
--- a/dom/canvas/WebGLFramebuffer.h	Thu Jun 11 06:37:35 2020 +0000
+++ b/dom/canvas/WebGLFramebuffer.h	Thu Jun 11 06:37:36 2020 +0000
@@ -10,6 +10,7 @@
 
 #include "mozilla/WeakPtr.h"
 
+#include "GLScreenBuffer.h"
 #include "WebGLObjectModel.h"
 #include "WebGLStrongTypes.h"
 #include "WebGLTexture.h"
@@ -138,6 +139,7 @@
   const GLuint mGLName;
   bool mHasBeenBound = false;
   const UniquePtr<gl::MozFramebuffer> mOpaque;
+  gl::SwapChain mOpaqueSwapChain;
   bool mInOpaqueRAF = false;
 
  private: